### DISCUSSION TITLE
Regarding Popover/Tooltip documentation

### DISCUSSION THREAD

**Discussion by TripathyC** [twbs/bootstrap#35584] [Q&A]
I think you guys did a fabulous job by writing a rocket science level documents which a developer like me do not understand. 

I want to know about **how to update part of popover content in BS5.1 once data is fetched by AJAX?** 

**Is there a simple example for this?** 

I have researched a lot for this & seems like all threads are ending at nowhere.

**Comment by Git-Harshit** [comment]
You can use JavaScript for this purpose, by using [popover.setContent](https://getbootstrap.com/docs/5.3/components/popovers/#methods) method available in Boostrap JS library.
There's an example on setting content using setContent below the methods table. Please see if this helps.
